Client/Frontend/Browser/Tab.swift

```diff
@@ -361,11 +363,9 @@ class Tab: NSObject {
         deleteWebView()
         deleteNewTabPageController()
         contentScriptManager.helpers.removeAll()
-        guard let appDelegate = UIApplication.shared.delegate as? AppDelegate else { return }
-        
-        let rewards = appDelegate.browserViewController.rewards
+
         if !PrivateBrowsingManager.shared.isPrivateBrowsing {
-            rewards.reportTabClosed(tabId: Int(rewardsId))
+            rewards?.reportTabClosed(tabId: Int(rewardsId))
```

Comment:
If its not a huge refactor (which I don't think it should be… but famous last words), can we move this to BVC's delegate that notifies when a tab is closed? That way we don't have to pass in a `BraveRewards` object into `Tab`